docs/cmd/inspect.md

```diff
@@ -45,19 +46,27 @@ containerlab inspect --all
 | 3 | srl02.yml  | srl01    | clab-srl01-srl  | 13c9e7543771 | srlinux:20.6.3-145 | srl  |       | running | 172.20.20.2/24 | 2001:172:20:20::2/80 |
 | 4 |            |          | clab-srl01-srl2 | 8cfca93b7b6f | srlinux:20.6.3-145 | srl  |       | running | 172.20.20.3/24 | 2001:172:20:20::3/80 |
 +---+------------+----------+-----------------+--------------+--------------------+------+-------+---------+----------------+----------------------+
+```
+
+#### Provide information about a specific running lab
+
+Provide information about the running lab named `srl02`
 
-# provide information about the running lab named srl02
-containerlab inspect --name srlceos01
+```bash
+❯ containerlab inspect --name srlceos01
 +---+---------------------+--------------+---------+------+-------+---------+----------------+----------------------+
 | # |        Name         | Container ID |  Image  | Kind | Group |  State  |  IPv4 Address  |     IPv6 Address     |
 +---+---------------------+--------------+---------+------+-------+---------+----------------+----------------------+
 | 1 | clab-srlceos01-ceos | 90bebb1e2c5f | ceos    | ceos |       | running | 172.20.20.4/24 | 2001:172:20:20::4/80 |
 | 2 | clab-srlceos01-srl  | 82e9aa3c7e6b | srlinux | srl  |       | running | 172.20.20.3/24 | 2001:172:20:20::3/80 |
 +---+---------------------+--------------+---------+------+-------+---------+----------------+----------------------+
 
+```
```

Comment:
here lets add an example where `inspect` command takes in a path to a topology file

Reply:
Example has been added we now have `Provide information about a specific running lab by its name` and `Provide information about a specific running lab by its topology file`